src/main/java/frc/robot/subsystems/auto/AutoLogic.java

```diff
@@ -293,4 +294,7 @@ public static Command isCollected() {
     }
     return Commands.none().withName("isCollected");
   }
+  public static Command readyIntakeCommand() {
+    return r.superStructure.coralPreIntake().withName("readyIntake");
```

Comment:
```suggestion
    if (r.superStructure != null) {
      return r.superStructure.coralPreIntake().withName("readyIntake");
    }
    return Commands.none().withName("readyIntake");
```
We should check that the superstructure isn't null before using it.